src/fanuc_sim/description/scene.xml

```diff
@@ -106,19 +110,58 @@
     <!-- Add a scene camera -->
     <site
       name="scene_camera_optical_frame"
-      pos="0.0 1.6 1.6"
-      euler="2.2415 0 3.1415"
+      pos="0.0 1.0 1.6"
+      euler="2.6415 0 3.1415"
     />
     <camera
       name="scene_camera"
-      pos="0.0 1.6 1.6"
+      pos="0.0 1.0 1.6"
       fovy="58"
       mode="fixed"
       resolution="640 480"
-      euler="-0.9 0 3.1415"
+      euler="-0.5 0 3.1415"
     />
-    <body name="arm_mount">
+    <body name="arm_pedestal" pos="0.0 0.3 0.15">
+      <geom type="box" size="0.2 0.2 0.15" />
+      <geom type="box" size="0.06 0.01 0.01" pos="0.23 0.03 0.1" />
+      <geom type="box" size="0.06 0.01 0.01" pos="0.23 -0.03 0.1" />
+      <site name="tool_holder_site" pos="0.25 0 0.15" euler="3.1415 0 0" />
+    </body>
+    <body name="arm_mount" pos="0.0 0.3 0.3">
       <include file="lrmate200id/lrmate200id.xml" />
     </body>
   </worldbody>
+
+  <!-- Define weld constraints between the robot flange (tool0) and the gripper, and between the suction cup (tool_tip)
+       and the blocks in the scene -->
+  <equality>
+    <weld
+      name="tool_attachment"
+      body1="tool0"
+      body2="gripper_base"
+      active="false"
+      torquescale="1"
```

Comment:
I think this needs to be increased. The attached tool is sort of "floppy" right now.